calculator/index.html

```diff
@@ -0,0 +1,42 @@
+<!doctype html>
+<html class="no-js" lang="">
+
+<head>
+    <meta charset="utf-8">
+    <title>Calculator App</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="../main.css">
+    <link rel="stylesheet" href="calc.css">
+    <link rel="icon" href="img/redi.png">
+</head>
+<body>
+<div class="headerBar">
+    <ul class="headerBar__links">
+        <li><a class="headerBar__link" href="/">Home</a></li>
+        <li><a class="headerBar__link headerBar__link--active" href="#">Calculator</a></li>
```

Comment:
This calculator link `href="#"` just opens the current url with `#` appended. example `localhost:5000/calculator/#` which is just the same page... Are you serving your local site via a webserver? I instructed the students to use Live Server vs code extention for example. or if you have node installed you can do `npx http-server .`

Reply:
I think we should be on the same page when we communicate to the students. Because if students to whom @BTBTravis told to use a web server will do that but the others just don't even know about such thing.

Reply:
It is my understanding that simply opening a HTML file via filesystem `file:///Users/t.shears/play/redi/fake_student_project/index.html` works differently on mac and windows. In my vs code setup I offered students use Live Server vs code extension to serve their files. Making thing the same on mac and windows and more in line with a real development setup.